core/src/main/java/org/apache/iceberg/SingleValueParser.java

```diff
@@ -335,6 +349,13 @@ public static void toJson(Type type, Object defaultValue, JsonGenerator generato
           generator.writeString(decimalValue.toString());
         }
         break;
+      case GEOMETRY:
+      case GEOGRAPHY:
```

Comment:
@szehon-ho, is the WKT representation worth adding a library to translate between WKT and WKB? Maybe this should do the same thing as binary like we do for variant?

Reply:
The JSON single-value serialization for geospatial types could only be used for initial/write default values. We found no real-world scenarios that require defining custom default values other than `null` for geospatial fields, so this WKT representation is not being used. We can leave JSON single-value serialization for geospatial values unimplemented and also remove JTS from iceberg dependency.

Reply:
If there aren't real-world scenarios for default values, then we should update the spec to ensure that the default is always null.

Reply:
Noted, will take a look at this